recipes/nbgrader/post-link.sh

```diff
@@ -0,0 +1,5 @@
+#{
+#  "${PREFIX}/bin/jupyter-nbextension" enable --sys-prefix --py nbgrader
+#  "${PREFIX}/bin/jupyter-serverextension" enable --sys-prefix --py nbgrader
+#} >>"$PREFIX/.messages.txt" 2>&1
+# XXX Placeholder for next nbgrader release
```

Comment:
I have no experience packaging `jupyter` extensions but if you do not need these files just remove them. (But I think you do. Maybe @bollwyvl can chime in because I believe he did a few of those.)